blackboxopt/optimizers/staged/bohb.py

```diff
@@ -49,29 +50,35 @@ def sample_around_values(
             samples agree on a value in a dimension.
         bw_factor: To increase diversity, the bandwidth is actually multiplied by this
             factor before sampling.
+        rng: A random number generator to make the sampling reproducible.
 
     Returns:
         Numerical representation of a configuration close to the provided datum.
     """
+    if rng is None:
+        rng = np.random.default_rng()
```

Comment:
```suggestion
        rng = np.random.default_rng(rng)
```
IIRC, `np.random.default_rng` will return the global instance if the provided seed is None, and simply the input if a valid generator is passed. So I think the `if` can be dropped here.